app/controllers/saml_idp_controller.rb

```diff
@@ -137,6 +137,7 @@ def capture_analytics
       # Logging to indicate if a validation bug fix will create a potentially breaking change
       analytics_payload[:encryption_cert_matches_matching_cert] =
         encryption_cert_matches_matching_cert?
+      analytics_payload[:cert_error_details] = saml_request.cert_errors
```

Comment:
i thought about using the existing `error_details` key, since it should be empty if the request is successful. but i thought that could be confusing ultimately, so decided not to -- open to thoughts about that!